package.json

```diff
@@ -128,6 +130,7 @@
     "nprogress": "0.2.0",
     "pixrem": "5.0.0",
     "playwright-chromium": "1.14.1",
+    "plop": "3.0.5",
```

Comment:
Kinda unfortunate that `plop` is 15MB https://packagephobia.com/result?p=plop

I guess we need it because it handles the multiple choice selection?

I was thinking we had something like this already 🤔

Reply:
Seems like a big portion of it is `inquirer` which we already have as a nested dependency and `@types/inquirer`